shuup/front/apps/personal_order_history/templates/shuup/personal_order_history/order_list.jinja

```diff
@@ -12,8 +12,8 @@
 {% endblock %}
 
 {% block dashboard_content %}
-    {% if orders %}
-        <div class="table-responsive">
+    {% if order_status_history %}
```

Comment:
oh no, show the order status changes inside the order detail instead 😅 , so the user can see the history changes for a single order